lib/Dialect/Verif/Transforms/PrepareForFormal.cpp

```diff
@@ -0,0 +1,69 @@
+//===- FlattenWires.cpp - Flattens all wires ------*- C++ -*-===//
```

Comment:
Nit: file name.

Reply:
(nit: after fixing filename, pad with dashes until hits 80chars like the others-- sometimes helps to just copy a different one next to this and pad until they line up 👍 )